src/alt-gosling-model/alt-text/color.ts

```diff
@@ -0,0 +1,83 @@
+import { GetColorName } from 'hex-color-to-color-name';
+
+// Converting colors to names
+
+export function getColorName(color: string, simple?: boolean): string {
+    console.log("color", "simple", color, simple);
+    color = color.replace('#', '');
+    if (simple === undefined || simple === false) {
+        return GetColorName(color).toLowerCase();
+    } else {
+        return findClosestColor(color);
```

Comment:
Maybe clarify what is the return value of the function by its name?:
```suggestion
function findClosestColorName(hex: string) {
```

Reply:
Yes, that's great! I'll add this in a new commit to make sure to capture all cases :)